administrator/components/com_media/resources/scripts/app/Api.js

```diff
@@ -106,16 +106,19 @@ class Api {
      * @TODO DN improve error handling
      */
     _handleError(error) {
-        alert(error.status + ' ' + error.statusText);
+        if (error.status !== 200) {
```

Comment:
Well, 200 status code did end up in _handleError and this is due to the `getimagesize($path)` in `libraries/vendor/joomla/image/src/Image.php`. Maybe handling that exception properly might fix this. Any ideas?
Thanks